sdk/communication/communication-administration/test/utils/recordedClient.ts

```diff
@@ -66,45 +66,34 @@ export const environmentSetup: RecorderEnvironmentSetup = {
 };
 
 export function createRecordedCommunicationIdentityClient(
-  context: Context
```

Comment:
Is this refactoring necessary? I prefer the previous code with two explicit functions that have specific names. Bool flags on functions to switch behavior are usually a code smell.